### PR TITLE
fix-issue-330.

### PR DESCRIPTION
**Pull request by vkamunya**
#### What is included in this PR?

- closes #330
- Inserted the correct screenshot for the web-ui login for the 'Launching the ODK-X Server' section.
- Revised instructions for starting/stopping the Stack/Swarm as requested and placed them in a subsection the same level as the Anonymous Access  section

### CODE REVIEW

**Comment by elmps2018** [comment]
Hi @vkamunya! Thanks for working on this! But it looks like you changed almost 5,000 files, so I am guessing something went astray...

**Comment by vkamunya** [comment]
Hi @elmps2018 , I'm not sure what went astray because i only worked on the sync-endpoint-cloud-setup.rst file. What could be the issue?

**Comment by elmps2018** [comment]
@vkamunya you can look into the "files changed" -- did you make sure to use the most recent version of docs as the starting point?

**Comment by vkamunya** [comment]
@elmps2018 I used the most recent version of the docs. From the files changed I see most of them are because of my pip installation and other installation packages, when i was working on it i was prompted to update most of the versions and i see how that could be an issue. Is there any way to change that?

**Comment by elmps2018** [comment]
@vkamunya that does look like the issue! I am not sure how to fix it though... maybe ask on Slack and see if anyone has suggestions?

**Comment by vkamunya** [comment]
I will, thanks!

**Comment by vkamunya** [comment]
Should I ask on the Outreachy channel or what channel should i ask on @elmps2018?

**Comment by elmps2018** [comment]
@vkamunya Outreachy channel is a good one hopefully someone there will know!

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 1]
```diff
@@ -577,16 +577,20 @@ Launching the ODK-X Server
   .. image:: /img/setup-azure/azure11.png
    :width: 600
 
-.. note::
-    If you are unable to log in, you may need to take the docker stack down and bring it back up again. That can be done with the following commands below:
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:581]: I think this text was helpful, good to keep.

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 2]
```diff
@@ -577,16 +577,20 @@ Launching the ODK-X Server
...
+.. _sync-endpoint-setup-take-stack-down:
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:580]: This seems like it needs a header for taking the stack/swarm down and bringing it back up (currently listing just taking it down).

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 3]
```diff
@@ -577,16 +577,20 @@ Launching the ODK-X Server
...
 
+Take the Stack/Swarm Down
+----------------------------------------------
+      
 .. code-block:: console
 
-    $ docker stack rm syncldap
+    $ docker stack rm syncldap 
+
+Bring the stack/swarm up with HTTPS support execute this command in the sync-endpoint-default-setup folder:
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:589]: This sentence seems to be missing some words. Maybe "In order to bring..."

**Review by elmps2018 — COMMENTED** [review]
Issue 330 also referred to an issue with the screenshot, that needs to be fixed as well to close the issue. 
@wbrunette if you can review as well since you created this that would be great.

**Comment by vkamunya** [comment]
I changed the screenshot too @elmps2018 it is not reflected on the file changes but it is visible in my editor.

File: odkx-src/sync-endpoint-cloud-setup.rst [hunk 4]
```diff
@@ -577,16 +577,19 @@ Launching the ODK-X Server
   .. image:: /img/setup-azure/azure11.png
    :width: 600
 
-.. _sync-endpoint-setup-take-stack-down:
+.. _sync-endpoint-setup-take-stack-down-and-bring-back-up:
 
-Take the Stack/Swarm Down
+Take the Stack/Swarm Down and Bringing it Back Up
```
Comment (elmps2018) [inline odkx-src/sync-endpoint-cloud-setup.rst:582]: In terms of tense, "Take the Stack/Swarm Down and Bring it Back Up" or "Taking the Stack/Swarm Down and Bringing it Back Up"

Comment (vkamunya) [reply odkx-src/sync-endpoint-cloud-setup.rst]: Done

**Review by elmps2018 — APPROVED** [review]

**Review by wbrunette — APPROVED** [review]

**Pull request merged**